lib/v0_bindings_extensions.rb

```diff
@@ -153,9 +173,17 @@ def valid_colors?
         invalid_colors.empty?
       end
 
+      def invalid_sets
+        sets.present? ? sets - Api::V0::HighlightsSwagger::VALID_SETS : []
```

Comment:
If `sets` is always an array, which I think is true, this can simplify to 

```ruby
sets - Api::V0::HighlightsSwagger::VALID_SETS
```

because

```
irb(main):001:0> [] - [1,2,3]
=> []
```

Reply:
sets can be null